src/amp-story-player-impl.js

```diff
@@ -133,7 +134,8 @@ export class AmpStoryPlayer {
 
   /** @public */
   buildCallback() {
-    this.stories_ = toArray(this.element_.querySelectorAll('a'));
+    initLogConstructor();
+    this.stories_ = toArray(this.querySelectorAll('a'));
```

Comment:
Shouldn't this be `this.element_` since we reverted the custom element PR?

Reply:
Yes, the merge conflict messed it up. Fixed it.